modules/template/services/ImportService.php

```diff
@@ -12,16 +12,17 @@
 use humhub\modules\custom_pages\modules\template\events\DefaultTemplateEvent;
 use humhub\modules\custom_pages\modules\template\models\Template;
 use humhub\modules\custom_pages\modules\template\models\TemplateElement;
-use humhub\modules\file\models\FileContent;
 use Yii;
 use yii\db\ActiveRecord;
 
-class ImportService
+/**
+ * Service to import Template
+ */
+class ImportService extends BaseImportService
```

Comment:
Same as Export class names, Maybe better `TemplateImportService` and `TemplateInstanceImportService`?

Reply:
https://github.com/humhub/custom-pages/pull/407#discussion_r2113808964